src/media/modules/allNodes.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+
+namespace WPML\PB\Elementor\Media\Modules;
+
+
+class allNodes extends \WPML_Elementor_Media_Node {
```

Comment:
@GeorgeBotsev, the class name should start with uppercase `AllNodes`.

Reply:
OK, fixing this now.